app/src/main/java/org/hisp/dhis/android/dataentry/reports/TeisNavigatorImpl.java

```diff
@@ -0,0 +1,26 @@
+package org.hisp.dhis.android.dataentry.reports;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+import android.support.v4.app.ActivityCompat;
+
+final class TeisNavigatorImpl implements ReportsNavigator {
```

Comment:
There are two ways to test navigators:
1) Through functional (UI) tests.
2) Using something like Roboelectric to mock out android components.

Personally, I prefer first approach. Unfortunately, due to lack of time because of MVP, we decided to temporarily abandon writing UI tests.